.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: CI
+
+on: [
+    push
+]
+
+jobs:
+  validation:
+    name: "Composer, CS, ArchSniffer"
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - php: '7.4'
+
+    env:
+      APPLICATION_ENV: devtest
+      APPLICATION_STORE: DE
+      PROJECT: after-pay
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+
+      - name: Configure sysctl limits
+        run: |
+          sudo swapoff -a
+          sudo sysctl -w vm.swappiness=1
+          sudo sysctl -w fs.file-max=262144
+          sudo sysctl -w vm.max_map_count=262144
+      - name: Composer get cache directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Composer cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+      - name: Composer validate
+        run: composer validate
+
+      - name: Composer install
+        run: |
+          composer --version
+          composer install
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: mbstring, intl, pdo_mysql
+          tools: composer:v2
+
+      - name: Codecept build
+        run: vendor/bin/codecept build --ansi
+
+      - name: Run CodeStyle checks
+        run: vendor/bin/phpcs -p -s --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml src/
```

Comment:
Ideally, the composer.json contains cs-check script, so here this can be used then instead.

Reply:
Transfered this code in composer.json